src/operators/Isovolume/avtIsovolumeFilter.C

```diff
@@ -8,21 +8,35 @@
 
 #include <avtIsovolumeFilter.h>
 
+#ifdef HAVE_LIBVTKH
+#include <vtkm/filter/ClipWithField.h>
+#include <vtkh/vtkh.hpp>
+#include <vtkh/DataSet.hpp>
+#include <vtkh/filters/IsoVolume.hpp>
+#include <vtkm/filter/CleanGrid.h>
+
+#include <vtkm/io/writer/VTKDataSetWriter.h>
```

Comment:
The includes of ClipWithField.h, CleanGrid.h, and VTKDataSetWriter.h, aren't necessary.

Reply:
Removed these lines.